openshift/templates/backup/backup-conf-configmap_DeploymentConfig.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
This file has one ugly name.

Reply:
Agree with the poor naming but have a feeling this is what the backup tool expects...
If not, we could rename it but not required for this PR to go thru

Reply:
I'd prefer to avoid renaming files copied from the backup-conf for traceability reasons.